package.json

```diff
@@ -179,7 +179,7 @@
   },
   "dependencies": {
     "@vscode/codicons": "^0.0.32",
-    "plist": "^3.1.0"
+    "plist": "github:TooTallNate/plist.js#pull/144/head"
```

Comment:
Your PR to the `plist.js` repo looks good to me. It makes sense to enable an `invalidCharReplacement` to be passed through to `xmlbuilder`.
I'm a little wary of having a dependency on a PR branch, as I'm not sure what happens when the PR branch is deleted (which is quite normal after a merge), even though that repo seems dead.

Do you think we can use [@expo/plist](https://www.npmjs.com/package/@expo/plist) instead? It looks better-maintained.

Reply:
I think it's okay to use the PR as a dependency. I tested it with a closed PR, and it still resolves correctly. The diffs continue to work as well.

I wasn't able to install the dependencies with [@expo/plist](https://www.npmjs.com/package/@expo/plist), it is also missing the latest merges in the original project.